tox.ini

```diff
@@ -2,14 +2,17 @@
 envlist = py27, py34, py35, py36, py37
 
 [testenv]
-deps = 
-    setuptools 
+deps =
+    setuptools
     Jinja2
     flexmock
     virtualenv-api
     virtualenv
     scripttest
     click
     spec2scl >= 1.2.0
-commands = python setup.py test --addopts -vv
+whitelist_externals = tar
+commands =
+    tar zcf tests/test_data/utest-0.1.0.tar.gz tests/test_data/utest
```

Comment:
Add the tar to gitignore? (If not already there.)

Reply:
Good idea.  Added.